server/service/src/sync/test/integration/errors.rs

```diff
@@ -30,57 +33,45 @@ mod tests {
         let mut service_provider = ServiceProvider::new(connection_manager.clone());
         struct TestService1(String);
         impl AppDataServiceTrait for TestService1 {
-            fn get_hardware_id(&self) -> Result<String, Error> {
+            fn get_hardware_id(&self) -> Result<String, std::io::Error> {
```

Comment:
Oops sorry this change is not needed, i move this logic and moved it back, sorry